web/testing/cypress/e2e/ID004_Create_Recurring_Event.cy.js

```diff
@@ -57,4 +57,35 @@ describe("Create Recurring Events", () => {
     cy.get("div").contains("COMP251").click(); // check that the event is there
     cy.get(".Content-title").contains("COMP251"); // check that the title is correct
   });
+
+  it("Create recurring event with wrong dates", () => {
+    const userEmail = `e2e+${Math.ceil(
+      Math.random() * 1000000000000000
+    )}@testdfas.com`;
+    console.log(userEmail);
+    cy.get(".w-full > .flex").click();
+    cy.origin(
+      "https://oncampus.us.auth0.com",
+      { args: { userEmail } },
+      ({ userEmail }) => {
+        cy.get("a").contains("Sign up").click();
+        cy.get("input#email").type(userEmail); //me might have to go through signup every time and create a new account? might trigger the auth0 limits if we do that...
+        cy.get("input#password").type("#e5PUJe9@mroL$", { log: false });
+        cy.contains("button[value=default]", "Continue").click();
+        cy.get("button").contains("Accept").click();
+      }
+    );
+
+    cy.visit("/schedule");
```

Comment:
Should check the "Repeat" box to make the event recurring

Reply:
oop you are right